spec/services/create_vehicle_spec.rb

```diff
@@ -0,0 +1,55 @@
+require 'rails_helper'
+
+describe CreateVehicle do
+  let(:params) do
+    {
+      brand: 'GM',
+      version: 'Camaro SS 6.2 V8 16V 406cv',
+      maker_year: '2016',
+      model_year: '2016',
+      license_plate: 'XXX1111',
+      city: 'Teresina',
+      uf: 'PI',
+      color: 'Preta',
+      fuel: 'Gasolina',
+      renavam: '123123123',
+      chassi: 'A123A123A123A123A123',
+      owner: 'Jhon Doe',
+      old_license_plate: nil
+    }
+  end
+
+  context 'save' do
+    it 'must be truthy' do
+      expect(CreateVehicle.create(params)).to be_truthy
+    end
+
+    it 'must be a vehicle object' do
+      expect(CreateVehicle.create(params)).to be_a Vehicle
+    end
+
+    it 'must be persisted' do
+      expect(CreateVehicle.create(params)).to be_persisted
+    end
+
+    context 'must return the vehicle created' do
+      let(:vehicle) { CreateVehicle.create(params) }
+
+      it { expect(vehicle.brand).to eq 'GM' }
+      it { expect(vehicle.version).to eq 'Camaro SS 6.2 V8 16V 406cv' }
+      it { expect(vehicle.maker_year).to eq 2016 }
+      it { expect(vehicle.model_year).to eq 2016 }
+      it { expect(vehicle.license_plate).to eq 'XXX1111' }
+      it { expect(vehicle.city).to eq 'Teresina' }
+      it { expect(vehicle.uf).to eq 'PI' }
+      it { expect(vehicle.color).to eq 'Preta' }
+      it { expect(vehicle.fuel).to eq 'Gasolina' }
+      it { expect(vehicle.renavam).to eq '123123123' }
+      it { expect(vehicle.chassi).to eq 'A123A123A123A123A123' }
+      it { expect(vehicle.owner).to eq 'Jhon Doe' }
+      it { expect(vehicle.old_license_plate).to be_nil }
+      it { expect(vehicle.status).to eq 'stock' }
```

Comment:
Assuming you are using the service, I think you don't need to test all these things… if I were you I could test only the status :P